benchmark/README.md

```diff
@@ -0,0 +1,37 @@
+# Keras Benchmarks
+
+## Overview
+The benchmakr module aims to provide a performance comparison on different Keras backends using various models and dataset.
+Currently supported backends: TensorFlow, MXNet 
+
+## Setup
+To switch between different backends refer to this documentation:
+
+We provide benchmark scripts to run on ImageNet DataSet and Synthetic Dataset(randomly generated) 
+### ImnageNet Dataset
+There are two training mode: train_on_batch and fit_generator, we recommend train_on_batch since it's more efficient on multi_gpu.
+Need to provide training mode, number of gpus and path to imagenet dataset.
+
+Example usage:
+
+`python benchmark_imagenet_resnet.py --train_mode train_on_batch --gpus 4 --data_path home/ubuntu/imagenet/train/`
+
+### Synthetic Dataset
+Scripts modified from [TensorFlow Benchmark](https://github.com/tensorflow/benchmarks/tree/keras-benchmarks/scripts/keras_benchmarks)
+
+Directly run the shell script to launch the benchmark, provide one of the configurations in config.json and whether you want to benchmark inference speed (True or False). 
+
+Example Usage:
+
+`sh run_<backend-type>_backend.sh gpu_config False`
+## Results
+Here we list the result on ImageNet and Synthetic Data(channels first) using ResNet50V1 model, on 1, 4 GPUs using AWS p3.8xLarge instance and 8 GPUs using AWS p3.16xLarge instance
+
+| GPUs   | ImageNet  | Synthetic Data(Channels First) |
```

Comment:
CPU?
Unit - Images/Sec
Please put a graph here. Mimic the same presentation we had in the document including setup table, results and graph. Graph is better to analyze scalability.

Reply:
Can you address these comments.